UnitsNet/UnitDefinitions/FoulingFactor.json

```diff
@@ -0,0 +1,68 @@
+{
+  "Name": "FoulingFactor",
+  "BaseUnit": "HourSquareFeetDegreefahrenheitPerBtu",
+  "XmlDoc": "Heat Transfer Coefficient or Thermal conductivity - indicates a materials ability to conduct heat.",
+  "Units": [
+    {
+      "SingularName": "HourSquareFeetDegreefahrenheitPerBtu",
+      "PluralName": "HourSquareFeetDegreefahrenheitsPerBtu",
+      "FromUnitToBaseFunc": "x",
+      "FromBaseToUnitFunc": "x",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "hr*ft2�F/Btu" ]
+        }
+      ]
+    },
+    {
+      "SingularName": "SquareMeterDegreeCentigradePerWatt",
+      "PluralName": "SquareMeterDegreeCentigradesPerWatt",
+      "FromUnitToBaseFunc": "x/5.6787",
+      "FromBaseToUnitFunc": "x*5.6787",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "m2�C/W" ]
+        }
+      ]
+    },
+    {
+      "SingularName": "SquareCentimeterKelvinPerWatt",
+      "PluralName": "SquareCentimeterKelvinsPerWatt",
+      "FromUnitToBaseFunc": "x/0.0005678",
+      "FromBaseToUnitFunc": "x*0.0005678",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "cm2K/W" ]
+        }
+      ]
+    },
+    {
+      "SingularName": "SquareCentimeterHourDegreeCentigradePerKilloCalorie",
```

Comment:
I got it from our domain expert and he confirmed above order is correct. Googled it, found many places used short form like:
Cm2Hr°C/kcal and hr*ft2°F/Btu or hr-ft2-°F/Btu.